source/jormungandr/jormungandr/scenarios/simple.py

```diff
@@ -185,9 +186,24 @@ def places(self, request, instance):
 
     def place_uri(self, request, instance):
         autocomplete = instance.get_autocomplete(request.get('_autocomplete'))
-        return autocomplete.get_by_uri(
-            uri=request["uri"], instances=[instance], current_datetime=request['_current_datetime']
-        )
+        try:
+            return autocomplete.get_by_uri(
+                uri=request["uri"], instances=[instance], current_datetime=request['_current_datetime']
+            )
+        except UnknownObject as e:
+            if not autocomplete.is_handling_stop_points():
+                # the autocomplete have not found anything and it does not handle stop_points,
+                # we'll search for the object in kraken too
+
+                kraken_res = instance.kraken_autocomplete.get_by_uri(
```

Comment:
Does this mean that we'll never be able to fully get rid of Kraken's autocompletion ?

Reply:
yup, at least for the /places/<uri>

(and maybe for the text search too for corner case, like sherbrooke where users explicitly search for stop_points)

Reply:
From afar:
* adding a stop_point index to mimir may be a thing to consider (and use/activate it only on those corner cases...)
* as for the reverse autocomplete isn't it doable to have an exact search on pt-ids, and coordinates of addresses?

Don't know the priority, doability, cost of all this.

Reply:
Maybe some day we will import stop_points and others pt_objects in bragi, but not now